ui/helpers/higher-order-components/authenticated/authenticated.container.js

```diff
@@ -12,4 +14,7 @@ const mapStateToProps = (state) => {
   };
 };
 
-export default connect(mapStateToProps)(Authenticated);
+export default compose(
+  withOptimisedRoute,
```

Comment:
This `Authentication` component is a route wrapper.

Unfortunately all routes inside the react-router-dom `<Switch>` statement are passed unstable the `exactMatch` property.

This HoC swallows this unstable property and prevents our components from rendering unnecessarily